dvc/utils/hydra.py

```diff
@@ -54,8 +54,12 @@ def compose_and_dump(
     with initialize_config_dir(config_dir, version_base=None):
         cfg = compose(config_name=config_name, overrides=overrides)
 
-    dumper = DUMPERS[Path(output_file).suffix.lower()]
-    dumper(output_file, OmegaConf.to_object(cfg))
+    suffix = Path(output_file).suffix.lower()
+    if suffix != ".yaml":
```

Comment:
```suggestion
    if suffix not in [".yml", ".yaml"]:
```

Not sure if we have a more robust way to check for YAML?

Reply:
Don't think there are more valid variations after `lower()` has been used above